dist/pom.xml

```diff
@@ -17,6 +17,12 @@
 
   <dependencies>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>unifiedconfig</artifactId>
```

Comment:
:wrench: I'm thinking just to name it as `config` or `configuration`. No need to have "unified" in the name.